cfgov/unprocessed/css/organisms/link-blob-group.less

```diff
@@ -20,6 +20,11 @@
     margin-left: -@grid_gutter-width / 2;
     margin-right: -@grid_gutter-width / 2;
   });
+
+  > h2 {
+        margin-left: @grid_gutter-width / 2;
+        margin-right: @grid_gutter-width / 2;
```

Comment:
Converted and realized this and the other `> h2` on the 50-50 text need to have the `@font-face` exposed to have the margin be sized correctly.